### ISSUE TITLE
'Move To Other AppVM' copies instead of moves

### ISSUE THREAD

**Issue by ohreally** [QubesOS/qubes-issues#3353]
#### Qubes OS version:

R4.0-rc3

#### Affected TemplateVMs: 

fedora-25

---

### Steps to reproduce the behavior:

Fresh install. Start file manager in personal domain (Domain; personal -> personal; Files). Create a file. Rightclick on the file, and select 'Move To Other AppVM...'.

### Expected behavior:

The file appears in the selected target AppVM, and disappears from the current VM.

### Actual behavior:

The file appears in the selected target AppVM, but does not disappear from the current VM.

**Comment by andrewdavidwong** [comment]
Do you know whether this affects only Fedora VMs?

**Comment by ohreally** [comment]
You're right: more details would be helpful. But i just installed Qubes for the first time yesterday, so I'm still discovering.

I created a VM based on Debian-8. But this does not have a filemanager included, so I ran qvm-move-to-vm in a terminal, and this went as expected: the file appeared 'there' and disappeared 'here'.
So, I went back to the 'personal' domain, based on Fedora-25, and ran the same command there, with the same (correct) result: there there, here gone.

So the problem has to be in the code for the right mouse menu in the filemanager.

Slightly related, and probably not worth creating another issue for:

The dialog which appears to confirm the action asks me to confirm operation qubes.Filecopy. This is a bit confusing, since I, as a layman user, would expect qubes.Filemove. Which probably doesn't even exist, but I am now asked to confirm an operation I did not request.

And the same dialog does not have the target VM selected, even when I run qvm-move-to-vm from the command line (which means that I have already specified the target).
